porch/api/porch/types_packagerevisions.go

```diff
@@ -302,3 +310,21 @@ type GitLock struct {
 	// This is set by kpt for bookkeeping purposes.
 	Commit string `yaml:"commit,omitempty" json:"commit,omitempty"`
 }
+
+type Condition struct {
+	Type string `yaml:"type" json:"type"`
+
+	Status ConditionStatus `yaml:"type" json:"status"`
+
+	Reason string `yaml:"type,omitempty" json:"reason,omitempty"`
```

Comment:
Nit: yaml:"type" -> reason

We also should just remove the yaml annotations - they aren't needed by the yaml libraries that apimachinery supports - but we can do that separately!

Reply:
Fixed the incorrect names, but we can clean up the tags in a separate PR.